### PR TITLE
switch to datawizard

### PR DESCRIPTION
**Pull request by mattansb**
instead of effectsize

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
do we need to reexport them? Can't we just use datawizard?

**Comment by IndrajeetPatil** [comment]
I don't think we need to re-export these functions from `{modelbased}`.

**Comment by mattansb** [comment]
But there are methods for both functions here. Will those work without also loading datawizard?

**Comment by strengejacke** [comment]
Not sure, I think we can conditionally include the namespace, like here:
https://github.com/easystats/performance/blob/master/R/pp_check.R (lines 120 following)

**Comment by IndrajeetPatil** [comment]
The builds are failing with this warning:

```r
❯ checking S3 generic/method consistency ... WARNING
Warning:   standardize:
    function(x, robust, two_sd, weights, verbose, ...)
  standardize.estimate_means:
    function(x, include_response, ...)
  
  standardize:
    function(x, robust, two_sd, weights, verbose, ...)
  standardize.estimate_predicted:
    function(x, include_response, ...)
  
  unstandardize:
    function(x, center, scale, reference, robust, two_sd, ...)
  unstandardize.estimate_predicted:
    function(x, include_response, ...)
  
  See section ‘Generic functions and methods’ in the ‘Writing R
  Extensions’ manual.

❯ checking Rd metadata ... WARNING
Warning:   Rd files with duplicated name 'reexports':
    ‘reexports.Rd’ ‘visualisation_recipe.estimate_predicted.Rd’
  Rd files with duplicated alias 'reexports':
    ‘reexports.Rd’ ‘visualisation_recipe.estimate_predicted.Rd’
```

Can you try just importing these functions from datawizard but not exporting them from modelbased?

```r
#' @importFrom datawizard standardize
#' @importFrom datawizard unstandardize
```

If these functions are called anywhere in the examples, then, yes, we will need to do `library(datawizard)`.

File: R/reexports.R [hunk 1]
```diff
@@ -2,3 +2,12 @@
...
 #' @importFrom datawizard visualisation_recipe
 datawizard::visualisation_recipe
+
+#' @export
+#' @importFrom datawizard standardize
+datawizard::standardize
+
+#' @export
+#' @importFrom datawizard standardize
+datawizard::unstandardize
```
Comment (IndrajeetPatil) [inline R/reexports.R:12]: There is a mistake here, btw.

It should be `#' @importFrom datawizard unstandardize`, and not `#' @importFrom datawizard standardize`

**Comment by strengejacke** [comment]
You need to define the S3 method differently in _datawizard_. 

If this is the generic:

`` r
standardize <- function(x,
                        robust = FALSE,
                        two_sd = FALSE,
                        weights = NULL,
                        verbose = TRUE,
                        ...) {
  UseMethod("standardize")
}
```

Than **all** other methods need to have _at least_ these arguments.

In _datawizard_, it should be:

`` r
standardize <- function(x, ...) {
  UseMethod("standardize")
}
```

**Comment by strengejacke** [comment]
Then, try this in _modelbased_:

https://github.com/easystats/modelbased/pull/169#issuecomment-1022045862

**Comment by mattansb** [comment]
> Warning:   Rd files with duplicated name 'reexports':
    ‘reexports.Rd’ ‘visualisation_recipe.estimate_predicted.Rd’
  Rd files with duplicated alias 'reexports':
    ‘reexports.Rd’ ‘visualisation_recipe.estimate_predicted.Rd’

What's that?

**Comment by DominiqueMakowski** [comment]
so we should change in datawizard first?

**Pull request merged**